pos_account_report_invoice_paid/views/report_invoice_templates.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <span
+            t-if="o.type == 'out_invoice' and o.state in ('open', 'in_payment', 'paid')"
+            position="replace">
+            <span t-if="o.type == 'out_invoice' and o.state in ('in_payment')">Invoice</span>
+            <span t-if="o.type == 'out_invoice' and o.state in ('open','paid')">Paid Invoice</span>
+        </span>
```

Comment:
We'll have a compatibility issue with `account_report_invoice_paid` module.

Both views are trying to replace the same element. One of them is going to fail (depending on the order they're loaded)

I think this module should depend on `account_report_invoice_paid` in order to properly extend the view and have both feature working.

What do you think?